test/tests/bootstrap/test_redfish10_os_bootstrap_parallel.py

```diff
@@ -0,0 +1,196 @@
+'''
+Copyright 2017, Dell EMC
+
+Author(s):
+George Paulos
+
+This script tests base case of the RackHD Redfish BootImage API and OS bootstrap workflows.
+This routine runs OS bootstrap jobs simultaneously on multiple nodes.
+For 8 tests to run, 8 nodes are required in the stack. If there are less than that, tests will be skipped.
+This test takes 15-20 minutes to run.
+
+OS bootstrap tests require special RackHD configuration and mirror repositories for the OS images.
+OS images are loaded via RackHD "httpProxies" settings in config.json.
+
+Example:
+      "httpProxies": [
+          {
+              "localPath": "/ESXi/5.5",
+              "remotePath": "/",
+              "server": "http://os-mirror-server/esxi/5.5/esxi"
+          },
+          {
+              "localPath": "/ESXi/6.0",
+              "remotePath": "/",
+              "server": "http://os-mirror-server/6.0/esxi6"
+          },
+          {
+              "localPath": "/CentOS/6.5",
+              "remotePath": "/",
+              "server": "http://os-mirror-server/centos/6.5/os/x86_64"
+          },
+          {
+              "localPath": "/CentOS/7.0",
+              "remotePath": "/",
+              "server": "http://os-mirror-server/centos/7/os/x86_64"
+          },
+          {
+              "localPath": "/RHEL/7.0",
+              "remotePath": "/",
+              "server": "http://os-mirror-server/rhel/7.0/os/x86_64"
+          }
+      ],
+
+For each OS type, the "localpath" string must conform to the convention above.
+The "server' URL points to the location of the OS executables in an image repository.
+
+'''
+
+import os
+import sys
+import subprocess
+
+# set path to common libraries
+sys.path.append(subprocess.check_output("git rev-parse --show-toplevel", shell=True).rstrip("\n") + "/test/common")
+import fit_common
+
+import flogging
+log = flogging.get_loggers()
+
+# This gets the list of nodes
+NODECATALOG = fit_common.node_select()
+
+# dict containing bootstrap workflow IDs and states
+NODE_STATUS = {}
+
+# download RackHD config from host
+rackhdconfig = fit_common.rackhdapi('/api/2.0/config')['json']
+rackhdhost = "http://" + str(rackhdconfig['apiServerAddress']) + ":" + str(rackhdconfig['apiServerPort'])
+
+
+# this routine polls a task ID for completion
+def wait_for_task_complete(taskid, retries=60):
+    for dummy in range(0, retries):
+        result = fit_common.rackhdapi(taskid)
+        if result['status'] != 200:
+            log.error(" " + result['text'])
+            return False
+        if result['json']['TaskState'] == 'Running' or result['json']['TaskState'] == 'Pending':
+            if fit_common.VERBOSITY >= 2:
+                print "OS Install workflow state: {}".format(result['json']['TaskState'])
```

Comment:
This part is not Stuart's new logging system so it should be converted to a log statement.
`if fit_common.VERBOSITY >= 2:
                print "OS Install workflow state: {}".format(result['json']['TaskState'])`